### DISCUSSION TITLE
Publish librespot 0.5

### DISCUSSION THREAD

**Discussion by hrkfdn** [librespot-org/librespot#1193] [General]
Hey dear librespot maintainers,

I work on ncspot which is a TUI frontend using librespot as a library. I was wondering whether there are plans to publish a 0.5 crate anytime soon? I'm following the development of librespot with a lot of interest and noticed that a bunch of fixes/features were merged that ncspot users could benefit from.

If not, are there blockers/issues in need of support?

Thanks for all your work on librespot
Henrik

**Comment by roderickvd** [comment]
I am no longer a Spotify user myself. I still have release rights. But depend on you guys to take on any issue reports flowing in. Once released, it will go to likely thousands of users, and this is a big release.

**Comment by hrkfdn** [comment]
Hey! Thanks for chiming in.
> Once released, it will go to likely thousands of users, and this is a big release.

Do you think this will improve over time as more of the past changes are "battle-tested" and matured or is it simply because a lot of changes have accumulated? If the latter is the case, wouldn't more frequent release cycles help with this? That said, I'm fully aware that everyone is doing this on their spare time, so please know I'm not trying to pressure anyone here.

If it matters: I have built ncspot with the `dev` branch of librespot and so far so good :)

**Reply by kingosticks** [reply]
The dev branch is broken for some people right now but I guess ncspot isn't using all parts of librespot. It's a good example of why it's not ready to release quite yet.

**Reply by hrkfdn** [reply]
That makes sense, thanks for clarifying.

**Comment by herrernst** [comment]
Related update: https://github.com/librespot-org/librespot/discussions/1203
